pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -514,23 +521,41 @@ func (mattermost *ClusterInstallation) GenerateDeployment(deploymentName, ingres
 		Value: valueSize,
 	})
 
-	// Mattermost License
-	volumeLicense := []corev1.Volume{}
-	volumeMountLicense := []corev1.VolumeMount{}
+	// Mattermost container volumes
+	volumes := []corev1.Volume{}
+	volumesMount := []corev1.VolumeMount{}
 	podAnnotations := map[string]string{}
+
+	// CA Certificate Files
+	volumesMount = append(volumesMount, corev1.VolumeMount{
+		Name:      CertificateFilesSecretName,
+		MountPath: CertificateFilesPath,
+		ReadOnly:  true,
+	})
+
+	volumes = append(volumes, corev1.Volume{
```

Comment:
this will be mounted only if we have a license, is this the behavior we want?

Reply:
That was the intention, yes. This way we don't even need to bother by checking if we have licenses.